src/media/js/views/homepage.js

```diff
@@ -6,6 +6,8 @@ define('views/homepage',
     var gettext = l10n.gettext;
     var appsModel = models('apps');
 
+    var $appContextmenu = document.getElementById('contextmenu').children[0];
```

Comment:
Yes, I use jQuery at first and switch to pure javascript later. Forgot to change the naming.